mail/src/test/java/javax/mail/internet/ParameterListTestSuite.java

```diff
@@ -57,7 +57,8 @@
     AppleFileNames.class,
     NonAsciiFileNames.class,
     DecodeParameters.class,
-    ParametersNoStrict.class
+    ParametersNoStrict.class,
+    ContentDispositionNoStrict.class
```

Comment:
The test can't be added to the ParameterListTestSuite because it only loads the
ParameterList class in a separate class loader and this test needs to load the
ContentDisposition class in a separate class loader.  Unfortunately, that means
creating a ContentDispositionTestSuite class.

It would also be good to add a ContentDispositionStrict test class.